tests/conftest.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from committeeoversight.wsgi import *
```

Comment:
hm, this feels a little unorthodox. can you set `DJANGO_SETTINGS_MODULE=committeeoversight.settings` in `setup.cfg` instead?

Reply:
Done!